frontend/src/Login/Login.jsx

```diff
@@ -1,9 +1,41 @@
 import './Login.css'
+import { useNavigate } from 'react-router-dom'
 
 export default function Login() {
   return (
-    <div>
+    <>
+      <div className='logincontainer'>
 
-    </div>
+        <div className='welcomestuff'>
+          <h1>Welcome Back</h1>
+          <p>Commited to serivce excellence</p>
+          <div className='socialmediaicons'>
+              <i className="fa-brands fa-facebook"></i>
+              <i className="fa-brands fa-twitter"></i>
+              <i className="fa-brands fa-instagram"></i>
+              <i className="fa-brands fa-youtube"></i>
+          </div>
```

Comment:
super cool integration of these apps! are we going to link them to the clinic's accounts?

Reply:
Most likely, yes.